spec/Plugin/QueryDefaultsPluginSpec.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace spec\Http\Client\Common\Plugin;
+
+use PhpSpec\Exception\Example\SkippingException;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\StreamInterface;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Psr\Http\Message\UriInterface;
+
+class QueryDefaultsPluginSpec extends ObjectBehavior
+{
+    public function it_is_initializable()
+    {
+        $this->beConstructedWith([]);
+        $this->shouldHaveType('Http\Client\Common\Plugin\QueryDefaultsPlugin');
```

Comment:
I think we use `::class` constants wherever possible. phpspec 3.0 generates specs like that by default.